### PR TITLE
Copy edit astropy documentation config package

### PR DESCRIPTION
**Pull request by lglattly**
A first pass at copy editing the documentation config package -- feedback always helpful!

### CODE REVIEW

**Comment by pllim** [comment]
Hmm this might conflict with @Cadair 's refactoring...

File: docs/config/index.rst [hunk 1]
```diff
@@ -65,22 +65,22 @@ Accessing Values
...
+configuration parameters. For instance, to get the default URL for
+``astropy`` remote data, do::
 
     >>> from astropy.utils.data import conf
     >>> conf.dataurl
     'http://data.astropy.org/'
 
-Changing Values at Run-time
+Changing Values at Runtime
 ---------------------------
```
Comment (pllim) [inline docs/config/index.rst]:
```suggestion
--------------------------
```

**Comment by bsipocz** [comment]
There is no changes affecting code here, so I'll cancel all but the html build.

**Comment by bsipocz** [comment]
All green, so merging. 

Thanks @lglattly!

**Pull request merged**